clients/fides-js/src/lib/tcf.ts

```diff
@@ -233,8 +242,16 @@ export const generateFidesString = async ({
         segments: [Segment.CORE],
       });
 
-      // Attach the AC string
-      const acString = generateAcString({ tcStringPreferences });
+      // Attach the AC string, which only applies to tcf_vendor_consents (no LI exists in AC)
```

Comment:
Are `userConsentedVendorIds` a subset of `disclosedVendorIds`? Trying to wrap my head around how to think about this relationship

Reply:
Yes. Basically the logic is that all of the disclosed vendors end up in the `dv.` list. However vendors included in the consent list should not be included in disclosed vendors list to reduce string length. So, for all intents and purposes, the DV list ends up being non-consented, disclosed.